curp-external-api/src/cmd.rs

```diff
@@ -34,7 +34,7 @@ pub trait Command:
     type ER: std::fmt::Debug + Send + Sync + Clone + Serialize + DeserializeOwned + PbSerialize;
 
     /// After_sync result
-    type ASR: std::fmt::Debug + Send + Sync + Clone + Serialize + DeserializeOwned;
+    type ASR: std::fmt::Debug + Send + Sync + Clone + Serialize + DeserializeOwned + PbSerialize;
```

Comment:
I think the `PbSerialize` is not a good name. The `Serialize` trait defines the `serialize` method while the `DeserializeOwned` defines the `deserialize` method. In `PbSerialize`, there are no methods named `serialize` or `deserialize`, only `encode` and `decode` methods. How about renaming `PbSerialize` to `PbCodec`?"

Reply:
Also, there is a typo in line 161. It's "Serialization", not "Serializaion"

Reply:
Makes sense. Renamed.